pkg/resizer/csi_resizer.go

```diff
@@ -36,6 +37,10 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	defaultFSType = "ext4"
```

Comment:
Given what we have discussed here: https://github.com/kubernetes-csi/external-provisioner/pull/400/files#diff-38b23914da9991d96c9ec4fc3c2525aaR76, should this also be configurable and allow default to be "none"?
@msau42 ^^

Reply:
For control-plane volume expansion it should *not* ideally matter. This exists mostly for being on same page as external-attacher , what we really want to pass is block vs fs type.

Reply:
Let me know if this is okay.  Feel free to remove the hold if you are fine with this.

Reply:
I think resizer should not have a defaultFSType.  Just keep "fsType := pvSpec.CSI.FSType" but don't set a default to be consistent with the proposed change in external-provisioner.  Otherwise we could potentially have a default fstype set to ext4 while driver internally use a different type.

In the external-provisioner, if we let driver set default to "none", driver will determine internally what fstype to use. 
Unfortunately this fstype is not returned by CreateVolumeResponse in CSI.  So pvSpec.CSI.FSType could be not set even if driver has used a non-ext4 type internally.  If we change pvSpec.CSI.FSType to ext4 in resizer in this case, that will be the wrong fstype.

Reply:
okay I have dropped default fstype from this PR.